experimental/packages/sdk-logs/src/index.ts

```diff
@@ -16,6 +16,7 @@
 
 export {
   LoggerConfig,
+  LoggerProviderConfig,
```

Comment:
Unrelated change?

Reply:
We should not really be importing from specific folders in a package `"@opentelemetry/sdk-logs/build/src/types"` as this makes it very fragile.

Reply:
I agree we should be avoiding it

Reply:
Oh end users. Ignore me